### ISSUE TITLE
Is there multi-language supports?

### ISSUE THREAD

**Issue by xohozu** [realm/jazzy#693]
A few days ago, I want to translate some existing awesome open-source framework's documentation into native language in-place. If keeping the original documentation words, jazzy will produce not friendly (or ugly) html pages. 

Does there exist some ways I missing to generate multi-language supports for jazzy, like source code have two version, Objective-C & Swift, which can switch quickly by a button. Or, it is a future feature.

##### Screenshots:
- source code
![image](https://cloud.githubusercontent.com/assets/2561942/20779626/9fe5c1b8-b7b0-11e6-82b5-5baed894fe9e.png)

- documentation
![image](https://cloud.githubusercontent.com/assets/2561942/20779608/7f5a1dcc-b7b0-11e6-96ac-84cdb91e2716.png)

**Comment by jpsim** [comment]
No, jazzy doesn't have the ability to split documentation into different language variants like you're proposing.

That'd be an amazing feature to add though! Would you be interested in building this functionality yourself, @xohozu?
